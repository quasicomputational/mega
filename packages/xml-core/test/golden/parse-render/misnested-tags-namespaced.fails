<a:bar xmlns:a="example:spam">
  <b:foo xmlns:b="example:eggs">
  Hello.
  </a:baz>
</a:bar>
