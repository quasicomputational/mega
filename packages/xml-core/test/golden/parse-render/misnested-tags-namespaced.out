Error: Misnested tags: Open tag is 'b:foo' at (2, 3), but close tag is 'a:baz' at (4, 5).
